cmd/kubeops/internal/handler/handler.go

```diff
@@ -109,6 +110,34 @@ func CreateRelease(cfg agent.Config, w http.ResponseWriter, req *http.Request, p
 	response.NewDataResponse(release).Write(w)
 }
 
+// OperateRelease decides which method to call depending on the "action" query param.
+func OperateRelease(cfg agent.Config, w http.ResponseWriter, req *http.Request, params handlerutil.Params) {
+	switch req.FormValue("action") {
+	case "upgrade":
+		upgradeRelease(cfg, w, req, params)
+	default:
+		// By default, for maintaining compatibility, we call upgrade.
+		upgradeRelease(cfg, w, req, params)
+	}
+}
+
+func upgradeRelease(cfg agent.Config, w http.ResponseWriter, req *http.Request, params handlerutil.Params) {
+	releaseName := params[nameParam]
+	chartDetails, chartMulti, err := handlerutil.ParseAndGetChart(req, cfg.ChartClient, requireV1Support)
```

Comment:
here `requireV1Support` is a bit of a readability issue. It seems that it's requiring v1 support. It would more readable if the var is called `requireV2Support` but it may be confusing if its value is `false`. Maybe `notRequireV1Support`?

Reply:
I was bugged by the exact same thought! But what do? Wouldn't it be just as confusing to write `const dontRequireV1Support = false`? Or would a comment suffice to mitigate that confusion?

Reply:
yes, a comment when setting the variable would be enough in my opinion

Reply:
I made an

Reply:
I felt naming the variable `dontRequireV1Support` or similar would make the code WET, since it would tie the constant name very closely to its value. I've renamed it to `isV1SupportRequired` instead; WDYT?